server/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# server/Dockerfile
```

Comment:
all comments must be in english please !

Reply:
ok ok